src/shared/components/case-editor/services/wizard-page-field-to-case-field.mapper.ts

```diff
@@ -40,9 +40,11 @@ export class WizardPageFieldToCaseFieldMapper {
     const caseFieldIds = override.complex_field_element_id.split('.');
     let case_field_leaf: CaseField;
 
-    if (this.isCollectionOfComplex(caseField)) {
+    let children = this.getCaseFieldChildren(caseField);
```

Comment:
should we consider removing the method isCollectionOfComplex since it's not being used anymore?

Reply:
Done